lib/scenes/scene_manager.cpp

```diff
@@ -15,6 +15,16 @@
 SceneManager::SceneManager(std::shared_ptr<EntityFactory> entity_factory, std::shared_ptr<EntityManager> entity_manager, BrickEngine* engine) : entity_factory(entity_factory), entity_manager(entity_manager), engine(engine) {};
 
 void SceneManager::loadLevel(Level& level) {
+    // Create the players
+    current_scene_entities.push_back(entity_factory->createGorilla(-300, -300, 1));
+    current_scene_entities.push_back(entity_factory->createPanda1(-300, -300, 2));
+    current_scene_entities.push_back(entity_factory->createPanda2(-300, -300, 3));
+    current_scene_entities.push_back(entity_factory->createPanda3(-300, -300, 4));
+    current_scene_entities.push_back(entity_factory->createWeapon(1000, 200, true));
+    current_scene_entities.push_back(entity_factory->createWeapon(1100, 200, false));
+    current_scene_entities.push_back(entity_factory->createWeapon(600, 200, true));
+    current_scene_entities.push_back(entity_factory->createWeapon(500, 200, false));
```

Comment:
The values are placeholders before they get spawned in the level which the user is about to play? 

Also, with how it is  done now, the players will be in the scene and get deleted. Isn't this the opposite to the idea we had about separating the players from the scene and 'recycling' them?

Reply:
Then a player needs a reset function to reset their position, health, sizes etc to default.